lib/mastalk/snippets/callout.html.erb

```diff
@@ -1,5 +1,5 @@
 # $~callout, ~$
 
 <div class='callout'>
-  <%= Mastalk::Document.new(body.strip).to_html %>
+  <%= Mastalk::Document.new(body.strip).to_html({ :auto_id => false }) %>
```

Comment:
Don't need the curly braces here (ruby doesn't require them if the last / only argument to a method is a hash).